### PR TITLE
Edited for flow/syntax/consistency

### PR DESCRIPTION
**Pull request by christinerose**
I've edited mostly for consistency and syntax flow / grammar, but I have some questions / comments before proceeding. 

1. It might be beneficial to define “Domains” and “Algebraic Effects” 
2. Seems definitions of Parallelism and Concurrency are very similar. Perhaps an example of the subtle difference, like not all Concurrent programs are Parallel, but all Parallel programs are Concurrent. 
3. Is it C API as in C / C++ API or CAPI as in (Common Application Programming Interface)?
4. The word "Domains" is sometimes capitalized and sometime not. Should it always be capitalized or always be lower case? Or are there instances where it's capitalized and others when it's lower case? Like `Domains.spawn` is capitalized, so I suspect it should always be capitalized. 
5. When talking about `dune`, does it need to be capitalized (or just in some cases). Should it be wrapped in `code block`?

Thanks! Looking forward to working on this further with you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -804,9 +802,9 @@ Index             Address  Node  PA cnt  records     Hitm    Total      Lcl
     1      0x7f2bf49a7b80     0     271      368    5.48%       76       76        0      123       76       47
 ```
 
-As evident from the report, there's quite a lot of false sharing happening in
-the code. To eliminate false sharing, we can allocate the Random state in the
-domain that is going to use it. This way, the states will be allocated with
+As evident from the report, there's quite a considerable amount of false sharing happening in
```
Comment (Sudha247) [inline README.md:805]: should we remove quite here?

**Review by Sudha247 — APPROVED** [review]
Thank you very much for the edits @christinerose! The changes look good to me.

To answer your questions:

>It might be beneficial to define “Domains” and “Algebraic Effects”

I agree, will add them.

>Seems definitions of Parallelism and Concurrency are very similar. Perhaps an example of the subtle difference, like not all Concurrent programs are Parallel, but all Parallel programs are Concurrent.

They may seem similar in the first glance, but there's a subtle difference as you mention. The wiki page expands on this a bit. On top of it, do you think a visual illustration will help?

>Is it C API as in C / C++ API or CAPI as in (Common Application Programming Interface)?

It is the former. C API refers to the [OCaml C Interface](https://ocaml.org/manual/intfc.html).

>The word "Domains" is sometimes capitalized and sometime not. Should it always be capitalized or always be lower case? Or are there instances where it's capitalized and others when it's lower case? Like Domains.spawn is capitalized, so I suspect it should always be capitalized.

The instance you quote is a code snippet. In the code snippet, `Domain` is an OCaml module, which is why the capitalization. Outside code snippets, `domain` is a term which refers to an independent unit of execution, much like `thread`. If it helps maintains consistency, I'm in favor of capitalizing it everywhere.

> When talking about dune, does it need to be capitalized (or just in some cases). Should it be wrapped in code block?

I think we need not capitalize dune everywhere. Dune stanzas that go into dune files are wrapped in code blocks.

File: README.md (hunk 1 above)
Comment (christinerose) [reply README.md:805]: That’s fine with me!

**Comment by christinerose** [comment]
They may seem similar in the first glance, but there's a subtle difference as you mention. The wiki page expands on this a bit. On top of it, do you think a visual illustration will help?

>> Either a visual illustration or perhaps a clarification on the subtle differences. ??
It might not be necessary for your audience. 

The instance you quote is a code snippet. In the code snippet, Domain is an OCaml module, which is why the capitalization. Outside code snippets, domain is a term which refers to an independent unit of execution, much like thread. If it helps maintains consistency, I'm in favor of capitalizing it everywhere.

>>In the code snipped, like when you write it as code, should that `Domain.spawn` be capitalized in Terminal or the Dune doc? If so, let’s keep it capitalized.
>>As for the rest, if “domain” is a term referring to an independent unit of execution, but it’s not a proper name of it, we’ll leave it lower-case. If it is a proper name, then we should capitalize. 
>>Here I’m looking for a way to visually differentiate between the two for clarity, if it’s necessary. If the command is always capitalized, let’s keep it capitalized and wrapped in the `code span`. 
>>Does this make sense?

When talking about dune, does it need to be capitalized (or just in some cases). Should it be wrapped in code block?
I think we need not capitalize dune everywhere. Dune stanzas that go into dune files are wrapped in code blocks.

>>For another project I talked about the capitalization of Dune with Jeremie and then Gemma. The overall consensus is that we capitalize the proper name of the program, like Dune stanzas or Dune 2.0, but anytime it’s talking about the command `dune`, it should be lower case and wrapped in that code block. Does that sound good?

**Comment by Sudha247** [comment]
>As for the rest, if “domain” is a term referring to an independent unit of execution, but it’s not a proper name of it, we’ll leave it lower-case. If it is a proper name, then we should capitalize.
Here I’m looking for a way to visually differentiate between the two for clarity, if it’s necessary. If the command is always capitalized, let’s keep it capitalized and wrapped in the code span.
Does this make sense?

That makes sense. In that case, we need not capitalize domains outside code blocks.

>For another project I talked about the capitalization of Dune with Jeremie and then Gemma. The overall consensus is that we capitalize the proper name of the program, like Dune stanzas or Dune 2.0, but anytime it’s talking about the command dune, it should be lower case and wrapped in that code block. Does that sound good?

Yep, we can stick with this.

**Pull request merged**